doc/Reactive/Reactive.02-general.md

```diff
@@ -0,0 +1,18 @@
+# General guidelines
+
+## API
+The main API provided by this package is [`IFeed<T>`](https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive/Core/IFeed.cs) which represents a stream of _data_.
+
+Unlike `IObservable<T>` or `IAsyncEnumerable<T>`, _feed_ streams are specialized to handle business data objects which are expected to be rendered by the UI.
+A _feed_ is a sequence of _messages_ which contains immutable _data_ with metadata such as its loading progress and/or errors raised while loading it.
+Everything is about _data_, this means that a _feed_ will never fail. It will instead report any error encountered with the data itself, remaining active for future updates.
```

Comment:
```suggestion
Everything is about _data_, which means that a _feed_ will never fail. It will instead report any error encountered with the data itself, remaining active for future updates.
```